spec/aerospike/client_spec.rb

```diff
@@ -22,6 +22,21 @@
 
   let(:client) { Support.client }
 
+  [Aerospike::Replica::MASTER, Aerospike::Replica::MASTER_PROLES, Aerospike::Replica::SEQUENCE, Aerospike::Replica::RANDOM].each do |replica_policy|
+  context "alternate #replica_policies" do
```

Comment:
Maybe you should include the current replica policy in the title, so that, when a spec fails, you can see which replica policy was in effect?

Reply:
Had it at first, but they are numbers. Kind of hesitant to write a `#to_s` just for tests. I'll just add the enum in the name.